docs/conf.py

```diff
@@ -1,5 +1,15 @@
+import os
+import sys
+
 import sphinx_rtd_theme
 
+cwd = os.getcwd()
+project_root = os.path.dirname(cwd)
+sys.path.insert(0, project_root)
+
+import cryptoconditions     # noqa
```

Comment:
is it really necessary to import cryptoconditions here?
This means that cryptoconditions needs to be installed in order to build the docs